src/fragments/lib/push-notifications/ios/getting_started/apns-cli-resources.mdx

```diff
@@ -9,6 +9,14 @@ Choose _APNS_ when promoted:
   SMS
 ```
 
+Follow the setup prompts: 
+```console
+? Provide your pinpoint resource name: › [your_name_here]
+? Apps need authorization to send analytics events. Do you want to allow guests
+ and unauthenticated users to send analytics events? (we recommend you allow this 
+ when getting started) (Y/n)
```

Comment:
@Equartey I would be happy if we can add a warning/quote about this in the documentation stating the importance of accepting this. When rejected user needs to be authenticated and if they update the auth later on, the current users might be deleted (at least they were for me).